budgetkey_data_pipelines/pipelines/entities/moital/pipeline-spec.yaml

```diff
@@ -0,0 +1,14 @@
+moital_service_providers:
+  pipeline:
+    - run: add_metadata
+      parameters:
+        name: moital_service_providers
+        title: Moital Service Providers
+    - run: scraper
```

Comment:
Let's add the service contractors as well, and concatenate the two into one big list.